### DISCUSSION TITLE
Learning outcomes sprint 2

### DISCUSSION THREAD

**Discussion by jabbarn** [Tunetonic/TuneTonic#40] [General]
please pick a one of the following topics (security, testing, design patterns and CI/CD) for your learning outcome during this sprint.

**Comment by jabbarn** [comment]
> please pick a one of the following topics (security, testing, design patterns and CI/CD) for your learning outcome during this sprint.

For sprint 2, I would like to improve the CI/CD with an expo integration for building an apk file for android.

**Comment by bcengioglu** [comment]
For this sprint, I want to write NestJS backend tests to check whether the requests are working as they should.

**Comment by koffhessel** [comment]
For sprint 2, I would like to improve security by adding JWT and protecting endpoints by checking these tokens on validity and role

**Comment by Imadbk7** [comment]
for sprint 2 i would like to test the features i made if they still work properly by writing tests for these features

**Comment by paldab** [comment]
Paul: For sprint 2 I would like to improve the security and regulations by researching the terms of conduct for deleting accounts and keeping them so we can create a application which complies with the rules.
